Dockerfile

```diff
@@ -1,27 +1,32 @@
 # Use the Node.js base image
-FROM node:latest
+FROM node:latest AS build
 
 # Set the working directory inside the container
 WORKDIR /app
 
+COPY package.json /app
+RUN mkdir /quartz && git clone https://github.com/jackyzha0/quartz.git /quartz && cd /quartz && npm install && npx quartz build
+RUN npm install
+
+
+FROM node:latest

```

Comment:
Niet echt nodig in dit geval, omdat alle data toch gecopieerd wordt, maar als voorbeeld.